bolt/lib/Rewrite/LinuxKernelRewriter.cpp

```diff
@@ -1305,6 +1334,27 @@ Error LinuxKernelRewriter::readAltInstructions() {
   return Error::success();
 }
 
+Error LinuxKernelRewriter::rewriteAltInstructions() {
+  // Disable output of functions with alt instructions before the rewrite
+  // support is complete.
+  for (BinaryFunction &BF : llvm::make_second_range(BC.getBinaryFunctions())) {
+    if (!BC.shouldEmit(BF))
+      continue;
+    for (const BinaryBasicBlock &BB : BF) {
+      if (!BC.shouldEmit(BF))
```

Comment:
Duplicate

Reply:
I think this might just be a typo and be `BB.shouldEmit()` instead maybe.

Reply:
It's not.